src/pages/HomeContent.jsx

```diff
@@ -577,25 +491,74 @@ export default function HomeContent() {
               expanded={step3open}
             />
           </StepContainer>
+          </div>
         ) : (
           <></>
         )}
+        <Container mobile={mobile} expert={difficulty == "DeFi Expert" ? true : false}>
+          { mobile ? <Items>
+            {alwaysShown.map((d) => {
+              return (
+                <Item key={d.title} notShown={false}>
+                  <Effect
+                    title={d.title}
+                    val={d.val}
+                    hasToolTip={d.hasToolTip}
+                    rewards={d.rewards}
+                  ></Effect>
+                </Item>
+              );
+            })}
+            {additionalDetails.map((d) => {
```

Comment:
Do we really want the stats to appear at the bottom in the "help me out" homepage view?

Reply:
![image](https://user-images.githubusercontent.com/43038510/215502628-7bdb2406-3823-4c9d-900c-240e2640417a.png)

Reply:
Honestly, maybe we should just take it out completely since people can view the stats on the Expert view...